### ISSUE TITLE
Remove name attributes from credit card fields

### ISSUE THREAD

**Issue by mindctrl** [restrictcontentpro/restrict-content-pro#1227]
We need to remove the name attribute from the card fields in 2Checkout, Authorize.net, PayPal Pro, and Stripe.

Ideally we remove the name attributes from all the credit card form templates and do something similar to what we do in Braintree. I'd like to standardize this if possible to avoid each gateway doing something unique.

Stripe recommends Elements or Checkout. No idea about Authorize.net. PayPal Pro... ugh.

**Comment by tnorthcutt** [comment]
@mindctrl Do the non-Braintree gateways currently use those name attributes in any way?

**Comment by mindctrl** [comment]
@tnorthcutt yes, they use it to validate that the fields aren't empty and then send that on to their APIs. See `RCP_Payment_Gateway_2Checkout::validate_fields()` for an example.

We should move to token type systems where possible, like we're doing in Braintree.
